src/client/resources/conversions.ts

```diff
@@ -508,6 +508,164 @@ export function createMigMigration(
   };
 }
 
+export function updateMigHook(
+  currentHook: any,
+  migHook: any,
+  currentPlanHookRef: any,
+  namespace: string,
+  currentPlan: any
```

Comment:
I think `currentPlan` could reuse the `IPlan` interface from the wizard `types.ts`, although maybe we should be defining some of those types in a more common place. These other hook `any` types should be replaced too if possible